cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/elements/areainterchange/CgmesTieFlow.java

```diff
@@ -0,0 +1,30 @@
+/**
+ * Copyright (c) 2021, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.cgmes.conversion.elements.areainterchange;
+
+/**
+ * @author Marcos de Miguel <demiguelm at aia.es>
+ */
+public class CgmesTieFlow {
```

Comment:
I'm not sure this object is useful. Maybe using directly `Terminal` is enough

Reply:
I think we should keep a TieFlow abstraction. The TieFlow identifier could be required for some later business process. Furthermore, there is an attribute that I think we should store for each TieFlow: `positiveFlowIn`, that states if the flow entering the terminal is also a flow entering the control area. From the CGMES documentation: 

_True if the flow into the terminal (load convention) is also flow into the control area. For example, this attribute should be true if using the tie line terminal further away from the control area. For example to represent a tie to a shunt component (like a load or generator) in another area, this is the near end of a branch and this attribute would be specified as false._

Reply:
About `positiveFlowIn`: I've seen it but I'm not sure how it is used? In any case, in TSOs I have access to, they are all set to `true`.

Do you have a particular business process in mind? Because for now, it does not have any additive value except of an ID that can be passed as an alias if we really need it...

Reply:
We prefer to keep the relevant information about the `TieFlow`: its identifier, the terminal identifier and the attribute for `positiveFlowIn`. This does not represent a big amount of information: there is a small number of control areas in each IGM (usually only one), each one having a relatively small number of tie flows (in the order of tens).

But if we really want to be strict on space we could drop and keep only the terminal identifier.

Reply:
The issue I have is more about readibility. It's not really intuitive to me to understand that `CgmesTieFlow` wrap IIDM `Terminal`... Do you have an opinion @annetill ?

Reply:
After some talking with Anne, I think we can start with using only `Terminal` and later a `CgmesTieFlow` object if we notice it is necessary

Reply:
Done